cli/daemon/run/testdata/echo/echo/echo.go

```diff
@@ -173,6 +174,9 @@ type AuthParams struct {
 
 //encore:authhandler
 func AuthHandler(ctx context.Context, params *AuthParams) (auth.UID, *AuthParams, error) {
+	if reflect.ValueOf(params).IsZero() {
```

Comment:
Pretty sure this just checks that it’s not a nil pointer. You need `.Elem().IsZero()`

Reply:
of course!